### ISSUE TITLE
Windows SDK 26100 is missing from the windows2022 image

### ISSUE THREAD

**Issue by fpribeiro** [actions/runner-images#10734]
### Description

The Windows 2022 image has Windows Driver Kit (WDK) 26100 but does not contain SDK 26100 (the latest SDK included is 22621).

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] Ubuntu 24.04
- [ ] macOS 12
- [ ] macOS 13
- [ ] macOS 13 Arm64
- [ ] macOS 14
- [ ] macOS 14 Arm64
- [ ] macOS 15
- [ ] macOS 15 Arm64
- [ ] Windows Server 2019
- [X] Windows Server 2022

### Image version and build link

windows-2022 20240929.1.1

### Is it regression?

No

### Expected behavior

Image contains SDK 26100 and WDK 26100

### Actual behavior

Image contains SDK 22621 and WDK 26100

### Repro steps

Build a project that requires APIs published in SDK 26100 and the build will fail.

**Comment by Prabhatkumar59** [comment]
Hi @fpribeiro - Thank you for bringing this issue to our attention. We are looking into this issue and will update you on this issue after investigating.

**Comment by dlech** [comment]
Related PR: https://github.com/actions/runner-images/pull/10452

**Comment by Usergitbit** [comment]
Running into this right now, currently preventing building the app. Also tried using this action https://github.com/GuillaumeFalourd/setup-windows10-sdk-action to install it but the build still fails and says it can not find it.

**Comment by Usergitbit** [comment]
It managed to get the sdk in the meanwhile installed using though it takes about 10,
```
           - name: Download Visual Studio installer
              run: |
                Invoke-WebRequest -Uri "https://aka.ms/vs/17/release/vs_enterprise.exe" -OutFile vs_enterprise.exe

            - name: Install WinSdk 26100 with VS
              run: |
                $process = Start-Process -FilePath vs_enterprise.exe -ArgumentList "modify", "--add", '"Microsoft.VisualStudio.Component.Windows11SDK.26100"', "--installPath", '"C:\Program Files\Microsoft Visual Studio\2022\Enterprise"', "--wait", "--quiet", "--force", "--norestart" -Wait -PassThru
                Write-Output $process.ExitCode

**Comment by Kichura** [comment]
The Windows 2022 SDK has been [deployed](https://github.com/actions/runner-images/commit/24b91c995e96c48dcf53349013670c0ab31285ff).

**Comment by RaviAkshintala** [comment]
@fpribeiro SDK 26100 is available in the Windows 22 Image version  `20241113.3`, We are closing this issue. Thanks,.
